Model/Sales/Repository/PackageRepository.php

```diff
@@ -201,10 +201,10 @@ public function isAllProductsFitIn($products, string $packageType): bool
         if ($packageType === AbstractConsignment::PACKAGE_TYPE_MAILBOX_NAME) {
             $mailboxProcent = $this->getMailboxProcent();
```

Comment:
Wat is dit voor setting? Als in, wat doet het? Procent is geen woord in het Engels, maar ik zie ook nergens docblocks waarin wordt uitgelegd waar het voor is. `mailboxPercentage` is in ieder geval al beter, maar ik denk dat er een accuratere naam voor bedacht kan worden... Is het veel moeite om te refactoren? 👼

Reply:
Ik heb er mailboxPercentage van gemaakt. (done)